### ISSUE TITLE
Hover state contrast to low

### ISSUE THREAD

**Issue by MarkGOC** [dto-btn/ssc-assistant#330]
![Image](https://github.com/user-attachments/assets/ca21c9b7-8c54-4031-b80e-8ba075f838d4)

Hover

![Image](https://github.com/user-attachments/assets/b1fe105d-c6cb-43ba-92bf-bfbe55ab7452)

When you hover over one of the buttons the contrast is to low.

We need to make it darker.  Or change our design approach for the hover state.  Maybe just add a dark border to the button on hover.  etc..

**Comment by GuillaumeTurcotte** [comment]
Should be fixed in dev, can you please test @MarkGOC . ty
